packages/jasmine/index.js

```diff
@@ -0,0 +1,23 @@
+// jasmine-core is a transitive dep of jasmine
+// so look under unhoisted location first
```

Comment:
this still needs some more commenting about why we have to do this special resolution instead of just getting the jasmine-core that node provides when we require it from here.

I think it's because we want to require() the same library that the user will later? why is it possible that a `require('jasmine-core')` in this location gives a different one than jasmine will later?

Reply:
Comment updated. PTAL.